smart-contracts/contracts-common/concordium-contracts-common/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased changes
+- Fixed `Display` trait on `VersionedModuleSchema` when module contained multiple contracts to render all of them.
+- Add `--all-features` to contracts common CI test execution.
```

Comment:
This last line should not be in the changelog, only relevant information for the users